.changeset/kind-weeks-kiss.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": patch
+---
+
+Add e2e label to be not run on every commit
```

Comment:
_E2e tests are no longer executed on every commit, this means performing e2e tests happens only when you add label "run pw-e2e"_